### ISSUE TITLE
Field request: trading name and alternative names

### ISSUE THREAD

**Issue by timgdavies** [openownership/data-standard#39]
At the moment [entity statements](http://beneficial-ownership-data-standard.readthedocs.io/en/master/schema.html#entitystatement) only have a ```name``` field.

This is defined as "The declared name of this entity."

There is user demand for other names, such as AKAs and Trading Names of a company.
 
There may also be a desire to identify when a company name is the legal name, branch name or something else.

**Comment by timgdavies** [comment]
The [OECD Common Reporting Standard for Automatic Exchange of Tax information](http://www.oecd.org/tax/automatic-exchange/common-reporting-standard/schema-and-user-guide/#d.en.345315) has the following set of values for it's nameType property:

```xml

<xsd:restriction base="xsd:string">
			<xsd:enumeration value="OECD201">
				<xsd:annotation>
					<xsd:documentation>SMFAliasOrOther</xsd:documentation>
				</xsd:annotation>
			</xsd:enumeration>
			<xsd:enumeration value="OECD202">
				<xsd:annotation>
					<xsd:documentation>indiv (individual)</xsd:documentation>
				</xsd:annotation>
			</xsd:enumeration>
			<xsd:enumeration value="OECD203">
				<xsd:annotation>
					<xsd:documentation>alias (alias)</xsd:documentation>
				</xsd:annotation>
			</xsd:enumeration>
			<xsd:enumeration value="OECD204">
				<xsd:annotation>
					<xsd:documentation>nick (nickname)</xsd:documentation>
				</xsd:annotation>
			</xsd:enumeration>
			<xsd:enumeration value="OECD205">
				<xsd:annotation>
					<xsd:documentation>aka (also known as)</xsd:documentation>
				</xsd:annotation>
			</xsd:enumeration>
			<xsd:enumeration value="OECD206">
				<xsd:annotation>
					<xsd:documentation>dba (doing business as)</xsd:documentation>
				</xsd:annotation>
			</xsd:enumeration>
			<xsd:enumeration value="OECD207">
				<xsd:annotation>
					<xsd:documentation>legal (legal name)</xsd:documentation>
				</xsd:annotation>
			</xsd:enumeration>
			<xsd:enumeration value="OECD208">
				<xsd:annotation>
					<xsd:documentation>atbirth (name at birth)</xsd:documentation>
				</xsd:annotation>
			</xsd:enumeration>
		</xsd:restriction>

```

**Comment by timgdavies** [comment]
Merged into #268
